### DISCUSSION TITLE
use shorter names like `s1reader`?

### DISCUSSION THREAD

**Discussion by yunjunz** [opera-adt/s1-reader#14] [Ideas]
First of all, I have really enjoyed using the reader since recently, burst-wise handling makes so much sense! 

As a user, the module name of `sentinel1_reader` seems too long to type, below is how I am using it now:

```python
from sentinel1_reader import sentinel1_reader as s1_reader, sentinel1_orbit_reader as s1_orbit_reader
```

Ideally, I would like to use the below to access everything this module has to offer, similar to [`s1etad`](https://gitlab.com/s1-etad/s1-etad) module.

```python
import s1reader
```

To this end, I am thinking of the following:

+ [x] 1. a repo structure as below:

```
s1-reader                 # repo name
    /src
        /s1reader         # module name
            reader.py
            burst_slc.py
            orbit.py
            etad.py
            ....
    /bin
    /docs
    ...
```

+ [x] 2. On the user side, it would be great to have the most commonly used functions available via `s1reader`, so that one could say `s1reader.burst_from_zip()` directly, without a "middle man" sub-modules. I don't have a clear idea of how to do that yet.

What do you think? @opera-adt/opera-adt

Update: implemented in https://github.com/opera-adt/s1-reader/pull/21.

**Comment by LiangJYu** [comment]
Sounds good to me. I like efficiency of your proposed rename.

**Comment by hfattahi** [comment]
I also like the shorter name you proposed.

**Comment by yunjunz** [comment]
Great, I will follow up with a PR for the renaming.

**Reply by yunjunz** [reply]
@hfattahi could you do the repo naming to `s1-reader`? This requires admin permission.

**Reply by hfattahi** [reply]
Done

**Reply by yunjunz** [reply]
Thanks!
